internal/logger/target/http/http.go

```diff
@@ -201,6 +201,7 @@ func (h *Target) logEntry(entry interface{}) {
 
 	resp, err := h.client.Do(req)
 	if err != nil {
+		atomic.StoreInt32(&h.connected, offline)
```

Comment:
Don't change a "closed" to offline.

```suggestion
		atomic.CompareAndSwapInt32(&h.connected, online, offline)
```